### ISSUE TITLE
Dropped PHP 7.3 compatibility ?

### ISSUE THREAD

**Issue by husim0** [symfonycorp/insight#100]
Hello,

Insight is not working anymore using PHP 7.3. Since the end of the last week apparently.

PHP 7.3 : 

```
$ docker run --rm -it php:7.3-cli-alpine sh
/ # apk add --no-cache curl
fetch https://dl-cdn.alpinelinux.org/alpine/v3.15/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.15/community/x86_64/APKINDEX.tar.gz
OK: 14 MiB in 32 packages
/ # curl -o insight.phar -s https://get.insight.symfony.com/insight.phar
/ # php insight.phar

Fatal error: Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 7.4.0". You are running 7.3.33. in phar:///insight.phar/vendor/composer/platform_check.php on line 24
/ # exit
```

```
$ docker run --rm -it php:7.4-cli-alpine sh
/ # apk add --no-cache curl
fetch https://dl-cdn.alpinelinux.org/alpine/v3.16/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.16/community/x86_64/APKINDEX.tar.gz
OK: 14 MiB in 33 packages
/ # curl -o insight.phar -s https://get.insight.symfony.com/insight.phar
/ # php insight.phar 
SymfonyInsight CLI 1.7.4 by Symfony (d4d92e6)

Usage:
  command [options] [arguments]

Options:
  -h, --help                       Display this help message
  -q, --quiet                      Do not output any message
  -V, --version                    Display this application version
      --ansi                       Force ANSI output
      --no-ansi                    Disable ANSI output
  -n, --no-interaction             Do not ask any interactive question
      --api-token=API-TOKEN        Your api token.
      --user-uuid=USER-UUID        Your user uuid.
      --api-endpoint=API-ENDPOINT  The api endpoint.
      --log[=LOG]                  Add some log capability. Specify a log file if you want to change the log location.
  -v|vv|vvv, --verbose             Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug

Available commands:
  analysis     Show the last project analysis
  analyze      Analyze a project
  configure    (Re-)Configure your credentials.
  help         Display help for a command
  list         List commands
  projects     List all your projects
  self-update  Update insight.phar to the latest version.
```

Thank you

**Comment by GrahamCampbell** [comment]
PHP 7.3 is EOL and dropping support for EOL PHP versions is typically not considered semver breaking. :)

**Comment by GrahamCampbell** [comment]
I don't think tags are what trigger updates to that phar. It looks like any push to `master`.

**Comment by nicolas-grekas** [comment]
Fixed in #101, thanks for reporting.
